src/shared/logger/activation.ts

```diff
@@ -198,3 +202,62 @@ async function registerLoggerCommands(context: vscode.ExtensionContext): Promise
         )
     )
 }
+
+/**
+ * Deletes the older logs when there are too many or are using too much space.
+ */
+async function cleanLogFiles(): Promise<void> {
+    let files = await fs.readdir(logPath)
+
+    if (files.length > 100) {
```

Comment:
Can we make '100', '10', and '2' named constants as well?
```js
MAX_LOG_FILES = 100
MAX_KEPT_LOG_FILES = 10
MIN_KEPT_LOG_FILES = 2
```
Your PR description gives these numbers meaning, so it'd be nice if the code reflected that.